docs/concepts/storage-consistency.rst

```diff
@@ -176,6 +176,28 @@ translog.
   a replica shard is accessed in terms of consistency. Only the refresh of the
   ``IndexReader`` affects consistency.
 
+.. NOTE::
```

Comment:
Instead of copying the content, I think it would be better to just link to the related content directly after the related claim at https://github.com/crate/crate/blob/master/docs/concepts/storage-consistency.rst?plain=1#L162.
E.g. 
`There is an exception to that when a lot of Primary Key values are detected inside a `WHERE` clause. See ... for details.`